test/api/unit/services/SQS.test.js

```diff
@@ -80,8 +80,6 @@ describe('SQS', () => {
     context("building a site's default branch", () => {
       it('should set an empty string for BASEURL in the message for a site with a custom domain', (done) => {
         const domains = [
-          'www.example.com',
```

Comment:
Got it. Okay, so we'll want to also stick a protocol onto all the domains without protocols when we work on the db.